package.json

```diff
@@ -22,53 +22,52 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-http#readme",
   "peerDependencies": {
-    "@angular/common": ">=4.3.6",
-    "@angular/core": ">=4.3.6",
-    "@blackbaud/auth-client": "^2.17.1",
+    "@angular/common": ">9.0.0-rc < 10.0.0",
+    "@angular/core": ">9.0.0-rc < 10.0.0",
+    "@blackbaud/auth-client": "^2.21.0",
     "@skyux/config": "^3.0.0"
   },
   "dependencies": {
     "@skyux/auth-client-factory": "1.1.0",
     "jwt-decode": "2.2.0"
   },
   "devDependencies": {
-    "@angular/animations": "7.2.15",
-    "@angular/common": "7.2.15",
-    "@angular/compiler": "7.2.15",
-    "@angular/compiler-cli": "7.2.15",
-    "@angular/core": "7.2.15",
-    "@angular/forms": "7.2.15",
-    "@angular/http": "7.2.15",
-    "@angular/platform-browser": "7.2.15",
-    "@angular/platform-browser-dynamic": "7.2.15",
-    "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.19.0",
-    "@pact-foundation/pact": "9.5.0",
-    "@pact-foundation/pact-web": "9.5.0",
-    "@skyux-sdk/builder": "3.11.0",
+    "@angular/animations": ">9.0.0-rc < 10.0.0",
```

Comment:
What's the reasoning for switching from explicit versions in `devDependencies`?

Reply:
Good point. I'll revert it back. I wanted the latest RC version, but this goes against our existing standards.